docs/07.DEBUGGER.md

```diff
@@ -65,9 +66,10 @@ All available commands of the client can be queried by the
 
 ## Integrating debugger support into applications using JerryScript
 
-The debugger can be enabled by calling the `jerry_debugger_init (uint16_t port)`
-function after the `jerry_init ()` function. It initializes the debugger
-and blocks until a client connects.
+The debugger can be enabled by calling the `jerryx_debugger_after_connect
+(jerryx_debugger_tcp_create (debug_port) && jerryx_debugger_ws_create ())`
+after the `jerry_init ()` function. It initializes the debugger and blocks
+until a client connects.
```

Comment:
It would be good to emphasize that users can choose from the available transport layer implementations provided by their own environments.